Build/Projects/FrameworkReferences.Net.definition

```diff
@@ -41,20 +52,27 @@
   </Platform>
 
   <Platform Type="Windows">
+    <Reference Include="System" />
+    <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Core" />
     <Reference Include="Lidgren.Network" />
   </Platform>
 
   <Platform Type="Windows8">
-    <Reference Include="MonoGame.Framework.Windows8" />
+    <Reference Include="System" />
+    <Reference Include="System.Runtime.Serialization" />
```

Comment:
Why are we adding `System` and `System.Runtime.Serialization` to every platform?  What changed?